flang/include/flang/Optimizer/Runtime/RTBuilder.h

```diff
@@ -14,28 +14,29 @@
 ///
 //===----------------------------------------------------------------------===//
 
-#ifndef FORTRAN_LOWER_RTBUILDER_H
-#define FORTRAN_LOWER_RTBUILDER_H
+#ifndef FORTRAN_OPTIMIZER_RUNTIME_RTBUILDER_H
+#define FORTRAN_OPTIMIZER_RUNTIME_RTBUILDER_H
 
-#include "flang/Lower/ConvertType.h"
+#include "flang/Common/Fortran.h"
 #include "flang/Optimizer/Builder/FIRBuilder.h"
 #include "flang/Optimizer/Dialect/FIRType.h"
 #include "mlir/IR/BuiltinTypes.h"
 #include "mlir/IR/MLIRContext.h"
 #include "llvm/ADT/SmallVector.h"
 #include <functional>
 
-// List the runtime headers we want to be able to dissect
-#include "../../runtime/io-api.h"
-
 // Incomplete type indicating C99 complex ABI in interfaces. Beware, _Complex
 // and std::complex are layout compatible, but not compatible in all ABI call
 // interface (e.g. X86 32 bits). _Complex is not standard C++, so do not use
 // it here.
 struct c_float_complex_t;
 struct c_double_complex_t;
 
-namespace Fortran::lower {
+namespace Fortran::runtime {
+class Descriptor;
+}
```

Comment:
Should we consider an abstraction layer sometime soon? We might want to at least consider what a move of runtime interfaces to `.td` files vs. using reflection on a set of C++ header files would look like.